hooks/tk-multi-publish2/basic/publish_node.py

```diff
@@ -0,0 +1,132 @@
+
+    @property
+    def description(self):
+        """
+        Verbose, multi-line description of what the plugin does. This can
+        contain simple html for formatting (:class:`str`)
+        """
+        return "Publishes all the outputs associated with this node."
+
+    @property
+    def settings(self):
+        """
+        Dictionary defining the settings that this plugin expects to recieve
+        through the settings parameter in the accept, validate, publish and
+        finalize methods.
+        A dictionary on the following form::
+            {
+                "Settings Name": {
+                    "type": "settings_type",
+                    "default": "default_value",
+                    "description": "One line description of the setting"
+            }
+        The type string should be one of the data types that toolkit accepts
+        as part of its environment configuration.
+
+        There are no settings for this item.
+
+        :returns: A dictionary of settings.
+        """
+        # inherit the settings from the base publish plugin
+        return {}
+
+    @property
+    def item_filters(self):
+        """
+        List of item types that this plugin is interested in.
+        Only items matching entries in this list will be presented to the
+        accept() method. Strings can contain glob patters such as *, for example
+        ["katana.*", "katana.session"]
+        """
+        return ["houdini.node.*"]
+
+    def accept(self, settings, item):
+        """
+        Method called by the publisher to determine if an item is of any
+        interest to this plugin. Only items matching the filters defined via the
+        item_filters property will be presented to this method.
+        A publish task will be generated for each item accepted here. Returns a
+        dictionary with the following booleans:
+
+        - accepted: Indicates if the plugin is interested in this value at
+          all. Required.
+        - enabled: If True, the plugin will be enabled in the UI, otherwise
+          it will be disabled. Optional, True by default.
+        - visible: If True, the plugin will be visible in the UI, otherwise
+          it will be hidden. Optional, True by default.
+        - checked: If True, the plugin will be checked in the UI, otherwise
+          it will be unchecked. Optional, True by default.
+
+        :param settings: Dictionary of Settings. The keys are strings, matching
+            the keys returned in the settings property. The values are `Setting`
+            instances.
+        :param item: Item to process.
+
+        :returns: dictionary with boolean keys accepted, visible, required and enabled.
+        """
+        # Naively accept all renders which have valid outputs
+        item.expanded = False
+        if not item._children:
+            return {"accepted": False}
+        return {"accepted": True, "checked": False}
```

Comment:
Switch around to avoid extra `not`

```suggestion
        if item._children:
            return {"accepted": True, "checked": False}
        return {"accepted": False}
```